api/internal/handler/route/route.go

```diff
@@ -309,7 +309,7 @@ func (h *Handler) Update(c droplet.Context) (interface{}, error) {
 		script, _ := h.scriptStore.Get(input.Route.ID)
 		if script != nil {
 			if err := h.scriptStore.BatchDelete(c.Context(), strings.Split(input.Route.ID, ",")); err != nil {
-				log.Warnf("script %s failed", input.Route.ID)
+				log.Warnf("delete script %s failed", input.Route.ID)
```

Comment:
I means do we should write the `err` into the error log, the error message is important.

Reply:
yes, It will write to the error log file.